source/environment.ts

```diff
@@ -16,6 +16,7 @@ export const environment = {
       (process.env.CARDANO_NETWORK as CardanoNetwork) || CardanoNetwork.MAINNET,
   },
   DEBUG: process.env.DEBUG,
+  GA_TRACKING_ID: 'UA-119953429-17',
```

Comment:
This tag should not appear anywhere in the codebase. No default value should exist

Reply:
You want to use it as proces.env variable?

Reply:
Ok, moved